docs/advanced-ping/slots.md

```diff
@@ -0,0 +1,61 @@
+# Slots
+
+We launched a new way for you to interact with your embeds, they're called Slots!

```

Comment:
I'd remove this line, since it is only relevant for a short period of time after launch.